docs/0-user-guides/1-etlas-user-guide/3-configuration/1-dependency-management.md

```diff
@@ -108,13 +108,11 @@ The `maven-repos` field can take a URL to a Maven repository OR a keyword from t
 | Keyword                    | Maven Repository URL                                     |
 | -------------------------- | -------------------------------------------------------- |
 | `central`                  | `https://repo1.maven.org/maven2/`                        |
-| `javaNet1`                 | `http://download.java.net/maven/1/`                      |
+| `javaNet1`                 | `https://maven.java.net/content/groups/public/`          |
```

Comment:
* maven/1/ is dead
* as is maven/2/

This url corresponds to maven/2/ -- I'm not sure if this is the right choice. My initial reaction was to just delete the line entirely.